src/app/auth/token-storage.service.ts

```diff
@@ -0,0 +1,60 @@
+import { Injectable } from '@angular/core';
+import  jwt_decode from 'jwt-decode';
+const TOKEN_KEY = 'AuthToken';
+const USERNAME_KEY = 'AuthUsername';
```

Comment:
these constants should be in a single util file, that contains all the constants for project

Reply:
defined clobal constants in common/AppConfig.ts